client/src/integrasjoner/api-utils.ts

```diff
@@ -1,4 +1,6 @@
 import { RestRessurs, RestStatus, Årsak } from "./rest-status";
+import { Size } from "../hooks/useWindowSize";
+import styles from "../komponenter/Infographic/Infographic.module.scss";
```

Comment:
Brukes disse to importene i denne filen?

Reply:
Fiksa